test/test_manifest.json

```diff
@@ -4243,6 +4243,18 @@
        "lastPage": 1,
        "type": "eq"
     },
+    {  "id": "issue12394",
+       "file": "pdfs/issue12392.pdf",
```

Comment:
This file is missing in the test folder. You'll need something similar to https://github.com/mozilla/pdf.js/pull/12361/files#diff-eba194d158906ef5ebd771295e400e6a.

Reply:
I'm reusing the test file for #12392:
https://github.com/mozilla/pdf.js/blob/master/test/pdfs/issue12392.pdf.link
So it should be ok, isn't it ? or need I to duplicate the pdf ?

Reply:
> So it should be ok, isn't it ?

Of course it's fine to re-use one PDF file for multiple tests, we already do that in other tests.

Reply:
I completely missed this since no file was added and the issue numbers looked similar at first sight. Reusing the file is definitely OK; sorry for the noise!